pkg/proxy/proxy.go

```
@@ -34,16 +42,39 @@ func NewProxyManager(dispatcher *dispatcher.Dispatcher) *Manager {
 }
 
 func (pm *Manager) AuthenticateRequest(req *http.Request) (*authenticator.Response, bool, error) {
-	c, err := req.Cookie(AuthProviderCookie)
+	cookie, err := req.Cookie(ObotAccessTokenCookie)
+	if err != nil {
+		return nil, false, nil
+	}
+
+	cookieOriginalValue := cookie.Value
+	cookieValue, err := url.QueryUnescape(cookie.Value)
```

are we ignoring error here?